Frameworks/UIKit/StarboardXaml/StarboardXaml.cpp

```diff
@@ -270,12 +270,16 @@ void UIApplicationBackgroundActivated(BackgroundActivatedEventArgs^ args) {
 
 void DoApplicationLaunch(ActivationType activationType, Platform::Object^ activationArg) {
     auto uiElem = ref new Xaml::Controls::Grid();
-    auto rootFrame = ref new Xaml::Controls::Frame();
-    rootFrame->Content = uiElem;
 
     XamlCompositor::Initialize(uiElem, activationType);
 
-    if (activationType != ActivationTypeLibrary) {
+    if (activationType == ActivationTypeLibrary) {
+        // In library mode, presented UI should completely cover whatever's behind it
+        uiElem->Background = ref new Xaml::Media::SolidColorBrush(Colors::White);
```

Comment:
nit: I know I pushed for this change, but now I'm wondering if we want a transparent background to support ViewControllers animating in/out?  We can probably worry about this when/if it becomes an issue, but it's up to you.